crates/rspack_binding_api/src/raw_options/raw_experiments/mod.rs

```diff
@@ -26,6 +26,7 @@ pub struct RawExperiments {
   pub cache: RawExperimentCacheOptions,
   #[napi(ts_type = "false | Array<RegExp>")]
   pub use_input_file_system: Option<WithFalse<Vec<RspackRegex>>>,
+  pub css: Option<bool>,
```

Comment:
This should be `bool`. RawOptions need to be normalized from the JS side already

Reply:
https://github.com/web-infra-dev/rspack/blob/0fe3eeae0645589b50f816fc8e52c6d4a83a488d/packages/rspack/src/config/defaults.ts#L217-L223

here , the default value of experiments.css is undefined . 

i think the undefined and false should have different semantics .


https://github.com/web-infra-dev/rspack/blob/0fe3eeae0645589b50f816fc8e52c6d4a83a488d/packages/rspack/src/config/normalization.ts#L329-L372

here ,  normalized config options , but it's not have experiments.css .   i am not sure whether i should change the default value of experiments.css to false here